arrow-fx-stm.api

```diff
@@ -0,0 +1,314 @@
+public abstract interface class arrow/fx/stm/STM {
```

Comment:
Hard remove the current only deprecation. Since this library is newly added it can go straight to 0.13.0 API.

Reply:
We should perhaps review the type names in this module for 0.13